src/Input/NumberInput.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import styled from "styled-components";
+
+import { colors } from "../shared/constants";
+
+const StyledContainer = styled.div`
+  width: 100%;
+  position: relative;
+  overflow-x: hidden;
+`;
+
+const StyledInner = styled.div`
+  width: 100%;
+  display: inline;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledTitle = styled.div`
+  float: left;
+  font-size: 18px;
+  font-weight: 600;
+  color: ${colors.DARK_GRAY};
+`;
+
+const StyledInputGroup = styled.div`
+  float: right;
+`;
+
+const StyledInput = styled.input`
+  width: 25px;
+  margin-left: 5px;
+  margin-right: 5px;
+  height: 20px;
+  outline: none;
+`;
+
+const StyledButton = styled.button`
+  width: 24px;
+  height: 24px;
+  outline: none;
+  border: ${`solid 1.5px ${colors.PURE_BLUE}`};
+  border-radius: 50%;
+  color: ${colors.PURE_BLUE};
+  opacity: ${props => (props.disabled ? "37%" : "100%")};
+`;
+
+interface NumberInputProps {
+  title: string;
+  value: string;
+  min: number;
+  max: number;
+  callback: (newValue: number) => void;
+  [key: string]: any;
+}
+
+export const NumberInput = ({
+  title,
+  value,
+  min = 0,
+  max = 99,
+  callback,
+  ...rest
+}: NumberInputProps) => (
+  <StyledContainer>
+    <StyledInner>
+      <StyledTitle>
+        <h3 style={{ display: "inline" }}>{title}</h3>
```

Comment:
Heading elements have semantic meaning and have implications for SEO, accessibility, etc. We should only use them when they are indeed used as "the third level heading".

Let's change this to something like a `div` or a `span`, even a `p` could do.

Also, let's not mix using styled-components and inline-styling as much as possible. Please stick to styled-components if you can. I also don't think `display: inline` is needed here if you go with flexbox.